src/utils/ContentRepositoryUtil.js

```diff
@@ -305,6 +316,7 @@ export default {
       Env: [
         "CONTENT_STORE_URL=" + repo.contentURL(),
         "CONTENT_STORE_APIKEY=supersecret",
+        "CONTENT_ID_BASE=" + repo.contentIDBase,
```

Comment:
:point_up: This was the key missing bit. Without a `_deconst.json` file, the preparers didn't actually submit content.